src/middlewared/middlewared/alert/base.py

```diff
@@ -245,8 +248,8 @@ class Alert:
     klass: type[AlertClass]
     args: typing.Union[dict[str, typing.Any], list]
     key: typing.Any
-    datetime: datetime
-    last_occurrence: datetime
+    datetime: DateTimeType
+    last_occurrence: DateTimeType
```

Comment:
Type annotation was being resolved as the member variable `datetime` rather than the imported module `datetime`.